pytorch_lightning/tuner/tuning.py

```diff
@@ -197,3 +246,34 @@ def lr_find(
         )
         self.trainer.auto_lr_find = False
         return result["lr_find"]
+
+
+def _check_tuner_configuration(
+    trainer: "pl.Trainer",
+    train_dataloaders: Optional[Union[TRAIN_DATALOADERS, "pl.LightningDataModule"]] = None,
+    val_dataloaders: Optional[EVAL_DATALOADERS] = None,
+    dataloaders: Optional[EVAL_DATALOADERS] = None,
+    method: str = "fit",
+):
+    supported_methods = ("fit", "validate", "test", "predict")
+    if method not in supported_methods:
+        raise MisconfigurationException(f"method {method!r} is invalid. Should be one of {supported_methods}.")
+
+    if method == "fit":
+        if dataloaders is not None:
```

Comment:
Could be a single condiiton statement

Reply:
then, if method is "fit" and dataloaders is None, it will fall under the else condition and will fail for the correct case I think